spec/serializers/user_serializer_spec.rb

```diff
@@ -511,17 +511,29 @@
         members_visibility_level: Group.visibility_levels[:owners],
       )
     end
-    let(:serializer) { UserSerializer.new(user, scope: Guardian.new, root: false) }
+    let(:serializer) { UserSerializer.new(user, scope: guardian, root: false) }
 
     before do
       group.add(user)
       group.save!
     end
 
-    it "should show group even when members list is not visible" do
-      json = serializer.as_json
-      expect(json[:groups].length).to eq(1)
-      expect(json[:groups].first[:id]).to eq(group.id)
+    context "when serializing user's own groups" do
+      let(:guardian) { Guardian.new(user) }
+
+      it "should show group even when members list is not visible" do
```

Comment:
word golf

I prefer this style of spec descriptions since it's shorter and more… definite 😃

```
      it "shows group even when members list is not visible" do
```